articles/openshift/concepts-networking.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Concepts - Networking diagram for Azure Red Hat on OpenShift 4
+description: Networking diagram and overview for Azure Red Hat OpenShift networking
+author: sakthi-vetrivel
+ms.author: suvetriv
+ms.topic: tutorial
+ms.service: container-service
+ms.date: 06/22/2020
+#Customer intent: As a cluster administrator, I want to understand networking in a Azure Red Hat OpenShift cluster.
+---
+
+# Networking in Azure Red Hat on OpenShift 4
+
+This guide covers an overview of networking in Azure Red Hat on OpenShift 4 clusters, along with a diagram and a list of important endpoints.
+
+For more details on core OpenShift networking concepts, please see the [Azure Red Hat OpenShift 4 networking documentation](https://docs.openshift.com/aro/4/networking/understanding-networking.html).
+
+## Networking Concepts in Azure Red Hat OpenShift
+
+### Networking Basics in OpenShift
+
+OpenShift Software Defined Networking (SDN) is used to configure an overlay network using Open vSwitch (OVS), an OpenFlow implementation based on Container Network Interface (CNI) specification. The SDN supports different plugins, and the plugin being used in Azure Red Hat on OpenShift 4 is Network Policy. All network communication is managed by the SDN, so no extra routes are needed on your virtual networks to achieve pod to pod communication.
+
```

Comment:
I believe that if you try to create the ARO cluster with Route Tables assigned on the subnets for Worker/Master, az aro create will fail.  Can we clarify this here to make sure that we state adding custom routes is not supported if that is true?